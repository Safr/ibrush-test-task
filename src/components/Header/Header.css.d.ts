export const Header: string;
export const header: string;
