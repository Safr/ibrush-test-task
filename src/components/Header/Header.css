.Header {
  padding: 10px;
  text-align: center;
  background-color: #bbe5f1;
  box-shadow: 20px 38px 34px -26px rgba(0,0,0,.2);
  border-bottom: 1px solid #41403e;
}

.Header h1 {
  margin: 0;
}