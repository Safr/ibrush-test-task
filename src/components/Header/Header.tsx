import React from 'react';
import classes from './Header.css';

const Header = () => {
  return (
   <header className={classes.Header}><h1>Exmo Api App</h1></header>
  );
};

export default Header;
