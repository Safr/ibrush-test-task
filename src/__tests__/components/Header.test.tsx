import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header/Header';

describe('Header Component', () => {
  test('should render correctly', () => {
    const wrapper = shallow(
      <Header />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
