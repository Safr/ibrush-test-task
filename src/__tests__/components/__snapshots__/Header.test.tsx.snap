// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render correctly 1`] = `
<header>
  <h1>
    Exmo Api App
  </h1>
</header>
`;
